### DISCUSSION TITLE
Discord hangout: 19 August 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4653] [Forum]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About
FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: August 19th, 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/aug-19-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
- What do we expect from pattern designers/programmers/maintainers? What can you expect as a pattern contributor?
- Tiers of pattern designs

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by joostdecock** [comment]
Oops, this one is in the future

**Comment by joostdecock** [comment]
Update name because I think `contributor call` sets the wrong expectations.

**Reply by tangerineshark** [reply]
FWIW, I think I'd rather tinker with what happens in the Contrib call to make it more focused than shift it to a Discord hangout. Granted, sometimes it is just a hang, especially because I've had a bit going on and haven't been able to do much pre-planning for the call. And maybe you've gotten feedback that this would be a better title, so I'm game to hear thoughts.

**Reply by eriese** [reply]
What's the wrong expectation that you think it sets?

**Reply by joostdecock** [reply]
To be clear: I'm not saying it *has to be* this name. I'm merely saying that a name change would help communicate that a course correction is taking place.

By *wrong expectations* I mean specifically that (some) people are under the impression the contributor call is a governance body. A misconception that has come to the fore recently in a rather unpleasant way and I'd rather preempt repetition of such antics.

More generally, there is tension inherent in the duality of having Discord being the *official* channel for support on one hand, and allowing people full agency over how they run the place (like we do for all social media platforms) on the other hand.

That is of course entirely my own doing, but hindsight is always 20/20.
Going forward I am resolving that tension by taking away the *official* part so Discord can do its own thing, just like Reddit, Facebook, Instagram, Twitter, ... do not have to concern themselves with whether or not a support channel should be named this or that (for example).

That's not the only reason, it also makes sense to not force our users in such a walled garden merely because they need some help.
